src/application/components/Request.js

```diff
@@ -5,6 +5,7 @@ import { cleanPath } from '../../utils';
 import JsonDoc from './JsonDoc/JsonDoc';
```

Comment:
jerry's suggestion to separate request parameters from request body in sections. Also do not display sections if they don't exist (i.e. no request body/no request parameters)